css/properties/text-underline-position.json

```diff
@@ -111,7 +111,7 @@
                 "notes": "See <a href='https://crbug.com/313888'>bug 313888</a>."
               },
               "chrome_android": {
-                "version_added": null
+                "version_added": false
```

Comment:
This shipped [in Chrome 71](https://storage.googleapis.com/chromium-find-releases-static/936.html#93625519dcf0a11cb77e9cd58a74d602d2307f47). Chrome and Android Webview should match. Please remove the note.